pgxn/neon_walredo/seccomp.c

```diff
@@ -9,6 +9,13 @@
  * To prevent this, it has been decided to limit possible interactions
  * with the outside world using the Secure Computing BPF mode.
  *
+ * This code is intended to support both x86_64 and aarch64. The latter
+ * doesn't implement some syscalls like open and select. So instead of
+ * open we use openat. We add both select (absend on aarch64) and
+ * pselect6 (present on both architectures) to the allowlist. Since we
+ * don't call select syscall directly we may expect that libc will call
+ * pselect6 on aarch64.
+ *
```

Comment:
We only use open/openat in the tests for whether seccomp is work properly. It's a pretty minor implementation detail, so I think this comment should be moved to a less prominent place. Maybe into  seccomp_load_rules() itself.

It's worth mentioning that we support x86_64 and aarch64, though, and that the system calls are different on different platforms. The set of system calls that we have whitelisted are the same on both platforms, but that's just accidental, not guaranteed.

Reply:
> The set of system calls that we have whitelisted are the same on both platforms

Hm, we use `select` which is not present on `aarch64` and comment talks about that. I'll try to rephrase.